spec.html

```diff
@@ -41522,6 +40971,7 @@ <h1>Properties of the AsyncFunction Prototype Object</h1>
         <li>is <dfn>%AsyncFunction.prototype%</dfn>.</li>
         <li>is an ordinary object.</li>
         <li>is not a function object and does not have an [[ECMAScriptCode]] internal slot or any other of the internal slots listed in <emu-xref href="#table-27"></emu-xref>.</li>
+        <li>is the value of the *"prototype"* property of %AsyncFunction%.</li>
```

Comment:
Why add this line, when we already have `is %AsyncFunction.prototype%`?

Reply:
with #2131, this line definitely isn't needed.